### ISSUE TITLE
throw new Error("Can't load yaml file");

### ISSUE THREAD

**Issue by StephenNay** [Redocly/openapi-cli#66]
I can certainly understand from this error message that the openapi-cli tool doesn't like something in my YAML file, but it would help a whole lot if it gave me some additional clue about what the problem is. Here is the entire error message:

```
    throw new Error("Can't load yaml file");
    ^

Error: Can't load yaml file
    at bundleToFile (/Users/v139708/.nvm/versions/node/v12.4.0/lib/node_modules/@redocly/openapi-cli/dist/bundle.js:32:11)
    at /Users/v139708/.nvm/versions/node/v12.4.0/lib/node_modules/@redocly/openapi-cli/dist/cli/index.js:70:55
    at Array.forEach (<anonymous>)
    at Command.<anonymous> (/Users/v139708/.nvm/versions/node/v12.4.0/lib/node_modules/@redocly/openapi-cli/dist/cli/index.js:67:17)
    at Command.listener (/Users/v139708/.nvm/versions/node/v12.4.0/lib/node_modules/@redocly/openapi-cli/node_modules/commander/index.js:291:8)
    at Command.emit (events.js:200:13)
    at Command.parseArgs (/Users/v139708/.nvm/versions/node/v12.4.0/lib/node_modules/@redocly/openapi-cli/node_modules/commander/index.js:672:12)
    at Command.parse (/Users/v139708/.nvm/versions/node/v12.4.0/lib/node_modules/@redocly/openapi-cli/node_modules/commander/index.js:459:21)
    at cli (/Users/v139708/.nvm/versions/node/v12.4.0/lib/node_modules/@redocly/openapi-cli/dist/cli/index.js:112:22)
    at Object.<anonymous> (/Users/v139708/.nvm/versions/node/v12.4.0/lib/node_modules/@redocly/openapi-cli/dist/index.js:45:20)
```
After getting this error, all I can do is guess as to what the problem is. My input YAML file (attached as a text file) is a bunch of $ref lines and some comments and not much more.

**Comment by StephenNay** [comment]
[NetSense_API_v3_source.test.yaml.txt](https://github.com/Redocly/openapi-cli/files/3990511/NetSense_API_v3_source.test.yaml.txt)

**Comment by StephenNay** [comment]
Sorry, that was a much-shortened version of the file that I created as an experiment, but it still produced the same error. The real version just has a lot more $ref statements in the `paths` section.

**Comment by knidarkness** [comment]
Hello, @StephenNay. Thank you for your attention and involvement. I will investigate your problem further, but the first note I would like to make is that

```yaml
components:
  $ref: ./schemas.yaml
  $ref: ./parameters.yaml
  $ref: ./securitySchemes.yaml
  $ref: ./responses.yaml
  $ref: ./requestBodies.yaml
  $ref: ./examples.yaml
```

is not a valid YAML document, as it has multiple $refs in a single node. Instead, it should be something like this:

```yaml
components:
  schemas:
    $ref: ./schemas.yaml
  parameters:
    $ref: ./parameters.yaml
  securitySchemas:
    $ref: ./securitySchemes.yaml
  responses:
    $ref: ./responses.yaml
  requestBodies:
    $ref: ./requestBodies.yaml
  examples:
    $ref: ./examples.yaml
```

However, what is actually to be fixed is the error message format and I'll check what we can do.

Or, if you would like, feel free to submit your PRs to help this project.

**Comment by StephenNay** [comment]
Thank you for pointing out the error in my YAML file. I had been using yamlinc and $include statements, and it doesn't have the same limitations as $ref. In order to try openapi-cli, I just replaced all the $include lines with $ref lines and didn't think about whether that broke any rules. I'll clean that up when I'm back at my desk in January.

**Comment by knidarkness** [comment]
The related #69 issue was resolved in #94 PR.

If you have any more questions or suggestions regarding, please reopen this issue, for now I'm closing it.
